tests/test_estimators/test_demographic_parity.py

```diff
@@ -4,7 +4,7 @@
 from sklearn.linear_model import LogisticRegression
 
 from sklego.common import flatten
-from sklego.linear_model import FairClassifier
+from sklego.linear_model import DemographicParityClassifier
```

Comment:
does it make sense to confirm that we have a deprecation warning of the deprecated model?

Reply:
Yes, will add that